forms.rst

```diff
@@ -389,349 +519,363 @@ object.
         }
 
 That's it! If you re-submit the form with invalid data, you'll see the
-corresponding errors printed out with the form.
+corresponding errors printed out with the form. Read the
+:doc:`Symfony validation documentation </validation>` to learn more about this
+powerful feature.
 
-Validation is a very powerful feature of Symfony and has its own
-:doc:`dedicated article </validation>`.
+Other Common Form Features
+--------------------------
 
-.. _forms-html5-validation-disable:
+Field Type Options
+~~~~~~~~~~~~~~~~~~
 
-.. sidebar:: HTML5 Validation
+Each :ref:`form type <form-types>` has a number of options to configure it.
+Many of these are specific to the field type and details can be found in
+the :doc:`Symfony form types reference </reference/forms/types>`. However, there
+are two common options that apply to most types: ``required`` and ``label``.
```

Comment:
There are _many_ options that apply to most types. I would slightly reword this "For instance, two commonly options options are: required and label.".